cmd2/cmd2.py

```diff
@@ -46,7 +46,7 @@
 from . import constants
 from . import plugin
 from . import utils
-from .argparse_completer import AutoCompleter, ACArgumentParser, ACTION_ARG_CHOICES
+from .argparse_custom import Cmd2ArgParser, CompletionItem
```

Comment:
Consider renaming `Cmd2ArgParser` to just `ArgParser` - in this context it is clear it is coming from `cmd2`, so the **Cmd2** prefix in the name is perhaps redundant.